morrf_ros/int16_image[] cost_maps
outputVals[] cost_values
morrf_ros/int16_image boundary_image
